components/tests/ui/robot_setup.sh

```diff
@@ -45,6 +47,34 @@ echo "A2,Treatment,10" >> "$BULK_ANNOTATION_CSV"
 # Create file for upload as File Annotation
 echo "Robot test file annotations" > "$FILE_ANNOTATION"
 
```

Comment:
so it is similar to the problem I reported here
https://trello.com/c/yc8YNhUa/117-omero-web-kills-running-script-on-logout
I think my comment was added before I discovered that issue